src/components/LoginPage.js

```diff
@@ -0,0 +1,66 @@
+import LoginBg from './LoginBg';
+import HeaderNav from './HeaderNav';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const LoginPage = () => {
+    
+    console.log('LoginPage has rendered.');
+
+    const [login, setLogin] = useState(false);
+
+    const handleClick = () => {
+        setLogin(!login);
+
+        // change url based on login state
+        if (login === false) {
+            navigate('/home-page');
+        } else if (login === true) {
+            navigate('/');
+        }
+    }
+
+    const navigate = useNavigate();
+
+    return (
+        <>
+            <header className="headerNav">
+                <div className="wrapper">
+                    <h1>NTS</h1>
```

Comment:
A nice touch might be if clicking the h1 takes the user to `/home-page` (or `/`, if not logged in), but fine either way.

Reply:
Thank you for the suggestion!! Implemented. :D